src/main/java/com/google/sps/task/Task.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.task;
+
+public final class Task {
+  private final String detail;
+  private final String keyString;
+  private final long timestamp;
```

Comment:
What time does this timestamp refer to? Can we give it a name to clarifies this?

Reply:
Oh this timestamp should refer to the time that the task was created. I would give it a name `creationTime` in the next commit.

Reply:
For this PR, I didn't modify the property name "timestamp" set for the task entity in datastore because I have already stored a lot of fake task entities with property name "timestamp". But in the next PR, I will modify the property name to "creationTime".

Reply:
Sounds good, thanks.